R/entry_functions.R

```diff
@@ -91,3 +91,33 @@ get_entry_picks <- function(entryid = NULL, gw = NULL){
   return(picks)
 }
 
+get_entry_captain <- function(entryid = NULL, gw = NULL){
+  if(is.null(entryid)) stop("You'll need to input an entry ID, mate.")
+  if(is.null(gw)) stop("You'll need to input a gameweek, mate.")
+  
+  captain3 <- data.frame()
+  for (j in 1:length(entryid)) {
+    captain2 <- data.frame()
+    gw_started <- get_entry(entryid[j])[["started_event"]]
+    gw_thisentry <- gw[gw >= gw_started]
```

Comment:
This version also includes a fix for managers that started their campaign in a later gameweek than what was requested.